Sources/Filters+Strings.swift

```diff
@@ -120,13 +120,23 @@ extension Filters {
     }
 
     static func removeNewlines(_ value: Any?, arguments: [Any?]) throws -> Any? {
-      let removeSpaces = try Filters.parseBool(from: arguments, index: 0, required: false) ?? true
+      let mode = arguments.first as? String ?? "all"
       guard let string = value as? String else { throw Filters.Error.invalidInputType }
 
-      let set: CharacterSet = removeSpaces ? .whitespacesAndNewlines : .newlines
-      let result = string.components(separatedBy: set).joined()
-
-      return result
+      switch mode {
+      case "all":
```

Comment:
Use some constants instead of magic strings here? Even if those constants are scoped inside the func.

e.g.
```swift
static func removeNewlines(_ value: Any? arguments: [Any?]) throws -> Any? {
  enum Modes: String {
    case all, leading
  }

  let modeArg = String(describing: arguments.first) ?? Modes.all.rawValue
  guard let mode = Modes(rawValue: modeArg) else {
    throw Filters.Error.invalidOption(option: modeArg)
  }

  switch mode {
  case .all: …
  case .leading: …
  }
}
```